config.default.json

```diff
@@ -4,9 +4,9 @@
       "serverAddress": "node1.goshimmer.dev:188"
     },
     "server": {
-      "port": 0
+      "bindAddress": "0.0.0.0:16178"
```

Comment:
@jonastheis do we need to update any test config files after that? Or does that not matter because it's disabled anyways?